src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -231,6 +234,29 @@ def register_source_uri_arguments(scope):
 for item in ['check-name', 'delete', 'list', 'show', 'show-usage', 'update', 'keys']:
     register_cli_argument('storage account {}'.format(item), 'account_name', account_name_type, options_list=('--name', '-n'))
 
+
+def replace_the_first(target, replacement):
+    def _wrap(arg):
+        arg.type.settings['help'] = arg.type.settings['help'].replace(target, replacement, 1)
+
+    return _wrap
+
+
+# Issue https://github.com/Azure/azure-cli/issues/1809 causes the parameter to be truncated at key
+# word 'Possible values include'. However the a few example of accept options in this command are
+# helpful.
+with ParametersContext('storage account list-account-sas') as c:
+    c.expand('parameters', AccountSasParameters, 'Account SAS', patches={
+        'permissions': replace_the_first('Possible values include', 'Choices include'),
+        'services': replace_the_first('Possible values include', 'Choices include')
+    })
+
+with ParametersContext('storage account list-service-sas') as c:
+    c.expand('parameters', ServiceSasParameters, 'Service SAS', patches={
+        'permissions': replace_the_first('Possible values include', 'Choices include'),
+        'resource': replace_the_first('Possible values include', 'Choices include')
```

Comment:
I would actually replace this SDK help text. Example from the `az storage account generate-sas` command:
```
--permissions    [Required]: The permissions the SAS grants. Allowed values: (a)dd (c)reate
                                 (d)elete (l)ist (p)rocess (r)ead (u)pdate (w)rite. Can be combined.
    --resource-types [Required]: The resource types the SAS is applicable for. Allowed values:
                                 (s)ervice (c)ontainer (o)bject. Can be combined.
    --services       [Required]: The storage services the SAS is applicable for. Allowed values:
                                 (b)lob (f)ile (q)ueue (t)able. Can be combined.
```
The important thing that the SDK text doesn't convey is that these values can be combined.